### ISSUE TITLE
Extension marked as Malware by Chrome Web Store

### ISSUE THREAD

**Issue by SpiderTechnitian** [ArshSB/DarkPDF#13]
Chrome web store no longer lists this extension, previously found at this url: https://chrome.google.com/webstore/detail/darkpdf/cfemcmeknmapecneeeaajnbhhgfgkfhp?hl=en-US

I received a popup from Chrome listing it as malware, saying it is disabled and suggests removal (which I obviously did)

Extension menu screenshot showing version 2.4.2 and the warning/recommendation: https://i.imgur.com/iTT5vF4.png

Please investigate.

**Comment by cruzzed** [comment]
me too on brave

**Comment by Jorge-D-Robles** [comment]
same here. Edge made me delete it as it was considered malware. looking at the source code, i don't see anything wrong with it, maybe a false flag?

**Comment by btarg** [comment]
I just got this warning from Brave, looks like the extension has been taken down. What kind of malware? are my accounts compromised? Reminder that just because the source is available here doesnt mean that a different, malicious version wasn't uploaded to the extension store

**Comment by acave-solweb** [comment]
So, [archive.org showed](https://web.archive.org/web/20240917175130/https://chromewebstore.google.com/detail/darkpdf/cfemcmeknmapecneeeaajnbhhgfgkfhp) the developer as "shilopron", and another extension ['Nucleus: A Pomodoro Timer and Website Blocker'](https://web.archive.org/web/20240923154957/https://chromewebstore.google.com/detail/nucleus-a-pomodoro-timer/koebbleaefghpjjmghelhjboilcmfpad) by the same dev is also unlisted now from the live webstore.

Plenty of possible speculation from malicious intent to a compromised account, but for now I'm keeping DarkPDF disabled and would recommend people do the same. **EDIT: I recommend just removing the extension completely, see below comment and associated links.**

Note: My disabled version is 2.4.2 published 4th September 2024, according to archive.org

**Comment by acave-solweb** [comment]
Additionally, the email on the chrome webstore came up in this: https://gist.github.com/c0m4r/45e15fc1ec13c544393feafca30e74de

This has a link to a blog:
https://palant.info/2024/10/30/the-karma-connection-in-chrome-web-store/

Don't know if actually related, but possibly hints at the authors webstore account being compromised at the least... (To speculate 😉)

I recommend just removing the extension. And I'm certainly looking closed at any future extensions I install...

**Comment by palant** [comment]
> Don't know if actually related, but possibly hints at the authors webstore account being compromised at the least...

The author’s account has not been compromised, this extension has simply been sold. The ownership transfer happened somewhere between June and September 2023, after that affiliate fraud and browsing profile collection functionality have been added to the extension. This repository belongs to the original author of the extension, not the new owners (presumably Karma Shopping Ltd).

My blog post contains more details: https://palant.info/2024/10/30/the-karma-connection-in-chrome-web-store/

**Comment by acave-solweb** [comment]
> The author’s account has not been compromised, this extension has simply been sold. The author change happened somewhere between June and September 2023, after that affiliate fraud and browsing profile collection functionality have been added to the extension. This repository belongs to the original author of the extension, not the new owners (presumably Karma Shopping Ltd).

Yeah after reading more into these posts I realise that's what's happened, selling out and putting users at risk sucks. I wonder if this repository will stay public much longer with this sorta attention..

**Comment by MuhtasimTanmoy** [comment]
What alternative available?

**Comment by btarg** [comment]
Time for someone to make an alternative. Hopefully one with more theming options!! Thanks for the insightful blogpost, @palant 
So much for keeping my browsing history private :sweat_smile:

**Comment by putskan** [comment]
> > Don't know if actually related, but possibly hints at the authors webstore account being compromised at the least...
> 
> The author’s account has not been compromised, this extension has simply been sold. The ownership transfer happened somewhere between June and September 2023, after that affiliate fraud and browsing profile collection functionality have been added to the extension. This repository belongs to the original author of the extension, not the new owners (presumably Karma Shopping Ltd).
> 
> My blog post contains more details: https://palant.info/2024/10/30/the-karma-connection-in-chrome-web-store/

Thanks! so using the extension from this source code should be ok?

**Comment by palant** [comment]
> Thanks! so using the extension from this source code should be ok?

I think so. The source code is merely outdated, not malicious.
